### PR TITLE
Docs: Add BigQuery migration to breaking changes doc

### PR DESCRIPTION
**Pull request by baldm0mma**
**What is this feature?**

Update the breaking changes doc with BigQuery migration path

**Why do we need this feature?**

So support isn't flooded with angry users

**Who is this feature for?**

All BigQuery community plugin data source users

**Which issue(s) does this PR fix?**:

Escalation [6201](https://github.com/grafana/support-escalations/issues/6201)

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [na] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by chrisharbro — APPROVED** [review]

File: docs/sources/breaking-changes/breaking-changes-v10-0.md [hunk 1]
```diff
@@ -204,6 +204,22 @@ Grafana v10.0.0 ships with the new React 18 upgrade. In turn, changes in the bat
...
+
+You've installed and are using the [DoiT International BigQuery data source plugin](https://github.com/doitintl/bigquery-grafana).
+
+#### Description
+
+Grafana v10.0.0 no longer supports the use of the [DoiT International BigQuery data source plugin](https://github.com/doitintl/bigquery-grafana), which was moved to a "retired" state in the latter half of 2022, and whose GitHub repo was archived December 2022. For these BigQuery data sources to continue functioning, users are required to migrate to the [Official Grafana BigQuery data source plugin](https://github.com/grafana/google-bigquery-datasource/).
+
+#### Migration path
+
+For everyone using Grafana 8.5+, it’s possible to import queries created with the DoiT International BigQuery community plugin by simply changing the data source to Grafana BigQuery in the data source selector in the affected panel. Please note that queries will be imported as raw SQL queries. More can be found [here](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin).
```
Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v10-0.md:219]:
```suggestion
For everyone using Grafana 8.5+, it’s possible to import queries created with the DoiT International BigQuery community plugin by simply changing the data source to Grafana BigQuery in the data source selector in the affected panel. Please note that [queries will be imported](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin) as raw SQL queries. 

#### Learn more 

[Information about importing DoiT International BigQuery queries](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin).
```

File: docs/sources/breaking-changes/breaking-changes-v10-0.md [hunk 2]
```diff
@@ -204,6 +204,22 @@ Grafana v10.0.0 ships with the new React 18 upgrade. In turn, changes in the bat
...
+
+**Note:**
```
Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v10-0.md:221]: Not sure if there's something missing here or if this isn't needed
```suggestion
```

File: docs/sources/breaking-changes/breaking-changes-v10-0.md [hunk 3]
```diff
@@ -204,6 +204,22 @@ Grafana v10.0.0 ships with the new React 18 upgrade. In turn, changes in the bat
 
 Update the plugin to version 1.8.3 or higher in your Grafana instance management console. This will ensure your plugin query editor works as intended.
 
+### Required update to Official Grafana BigQuery plugin; DoiT International BigQuery plugin no longer supported
```
Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v10-0.md:207]: Can we just go with a shorter heading? 
```suggestion
### DoiT International BigQuery plugin no longer supported
```

**Review by imatwawana — COMMENTED** [review]
I've made some small structural changes and left a couple questions.

**Comment by baldm0mma** [comment]
@imatwawana Thanks for the changes/catches. Should now be updated with suggestions.

File: docs/sources/breaking-changes/breaking-changes-v10-0.md [hunk 4]
```diff
@@ -204,6 +204,24 @@ Grafana v10.0.0 ships with the new React 18 upgrade. In turn, changes in the bat
...
+
+You've installed and are using the [DoiT International BigQuery data source plugin](https://github.com/doitintl/bigquery-grafana).
+
+#### Description
+
+Grafana v10.0.0 no longer supports the use of the [DoiT International BigQuery data source plugin](https://github.com/doitintl/bigquery-grafana), which was moved to a "retired" state in the latter half of 2022, and whose GitHub repo was archived December 2022. For these BigQuery data sources to continue functioning, users are required to migrate to the [Official Grafana BigQuery data source plugin](https://github.com/grafana/google-bigquery-datasource/).
+
+#### Migration path
+
+For everyone using Grafana 8.5+, it’s possible to import queries created with the DoiT International BigQuery community plugin by simply changing the data source to Grafana BigQuery in the data source selector in the affected panel. Please note that [queries will be imported](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin) as raw SQL queries.
```
Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v10-0.md:219]:
```suggestion
For everyone using Grafana v8.5+, it’s possible to import queries created with the DoiT International BigQuery community plugin by simply changing the data source to Grafana BigQuery in the data source selector in the affected panel. Please note that [queries will be imported](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin) as raw SQL queries. 
```

File: docs/sources/breaking-changes/breaking-changes-v10-0.md [hunk 5]
```diff
@@ -204,6 +204,24 @@ Grafana v10.0.0 ships with the new React 18 upgrade. In turn, changes in the bat
...
+
+Grafana v10.0.0 no longer supports the use of the [DoiT International BigQuery data source plugin](https://github.com/doitintl/bigquery-grafana), which was moved to a "retired" state in the latter half of 2022, and whose GitHub repo was archived December 2022. For these BigQuery data sources to continue functioning, users are required to migrate to the [Official Grafana BigQuery data source plugin](https://github.com/grafana/google-bigquery-datasource/).
+
+#### Migration path
+
+For everyone using Grafana 8.5+, it’s possible to import queries created with the DoiT International BigQuery community plugin by simply changing the data source to Grafana BigQuery in the data source selector in the affected panel. Please note that [queries will be imported](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin) as raw SQL queries. 
+
+#### Learn more 
+
+[Information about importing DoiT International BigQuery queries](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin).
```
Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v10-0.md:223]:
```suggestion
- [Information about importing DoiT International BigQuery queries](https://github.com/grafana/google-bigquery-datasource#importing-queries-created-with-doit-international-bigquery-datasource-plugin).
```

**Review by imatwawana — APPROVED** [review]
Added suggestions for a few nits that I missed on the first pass, but otherwise LGTM!

**Pull request merged**